multicast/recv.py

```
@@ -211,6 +212,82 @@
+_mcast_recv_just_bind_code = """
+sock = multicast.genSocket() if isock is None else isock
+sock.bind((multicast._MCAST_DEFAULT_GROUP, port))
+
+"""
+
+
+_mcast_recv_bind_not_join_msg = "...".join([
+	"Consider using",
+	_mcast_recv_just_bind_code,
+	"instead, for improved performance. Otherwise specify the multicast bind group.",
+])
```

Ruff (0.8.2)

Consider f-string instead of string join

Replace with f-string

(FLY002)